ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -218,6 +219,7 @@ export const NavBar = withUserProfile()(
           }
         </div>
         <Breadcrumb/>
+        <AccessRenewalNotificationMaybe profile={this.props.profile}/>
```

Comment:
The logic about whether this should appear or not is inside this component, so it decides if it should render

Reply:
I like the "Maybe" suffix to label this behavior.